src/components/Nav/Nav.js

```diff
@@ -1,13 +1,57 @@
 import React, { Component } from 'react';
+import ListContents from './component/ListContents';
 import './Nav.scss';
 
 export class Nav extends Component {
+  constructor() {
+    super();
+    this.state = {
+      productList: false,
+      listItem: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('/data/menuList.json', {})
```

Comment:
- fetch 함수의 2번째 인자가 없다면 그냥 생략해주세요!
```suggestion
    fetch('/data/menuList.json')
```

Reply:
넵!! 다른 fetch 함수들도 수정 완료했습니다!!